test/contrib/k8s_job_test.py

```diff
@@ -51,6 +56,15 @@ class SuccessJob(KubernetesJobTask):
         }]
     }
 
+    def signal_complete(self):
```

Comment:
Please remove this form this test, and test in a separate test case (SignalCompleteJob).